index.html

```diff
@@ -36,7 +37,129 @@ <h1>I’m Alexandra.<br>Glad to see you!</h1>
                             <li><a href="#"><img src="img/github.jpg" alt="github icon"></a></li>
                             <li><a href="#"><img src="img/angellist.jpg" alt="angellist icon"></a></li>
                         </ul>
-        </section>  
+        </section>
+    </section>  
+            <!-- the first card............... -->
+            <section id="work-section">
+                <div class="cards">
+                    <img src="img2/Snapshoot Portfolio.png" alt="portfolio img">
+                    <div class="align-text">
+                    <h2 class="h2-size">Tonic</h2>
+                    <ul class="first-list">
+                        <li class="list1">CANOPY</li>
+                        <li class="list2">Back End Dev</li>
+                        <li class="list2">2015</li>
+                    </ul>
+                    <P class="work-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                    <ul class="blue-list">
+                        <li>HTML</li>
+                        <li>Css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="work-btn btn">see project</button>
```

Comment:
[OPTIONAL] Let's make this button nicer 😄 how about we make sure that it follows the button styling: Active, Hover, and Pressed (see design) 👇🏽 

![Screenshot 2023-04-30 at 00 22 55](https://user-images.githubusercontent.com/45739963/235335411-e869e15b-5271-4c37-b112-37ddd9ce2c87.png)

You're applying the hover actually but you're not updating the text color when the hover is hovered, take a look 👇🏽 

![Screenshot 2023-04-30 at 00 20 48](https://user-images.githubusercontent.com/45739963/235335426-364fbc34-c438-474f-9fb9-21b2b3367e84.png)

Also, make sure to adjust the background color when it's not hovered, it's supposed to be white 👇🏽 

![Screenshot 2023-04-30 at 00 25 23](https://user-images.githubusercontent.com/45739963/235335461-ed02d8fe-609b-4795-a33c-787864fa4189.png)

Make sure to apply that to all the other buttons 👀